common/persistence/visibility/store/sql/query_converter_mysql.go

```diff
@@ -253,3 +256,32 @@ func (c *mysqlQueryConverter) buildSelectStmt(
 		searchattribute.GetSqlDbColName(searchattribute.RunID),
 	), queryArgs
 }
+
+func (c *mysqlQueryConverter) buildCountStmt(
+	namespaceID namespace.ID,
+	queryString string,
+) (string, []any) {
+	var whereClauses []string
+	var queryArgs []any
+
+	whereClauses = append(
+		whereClauses,
+		fmt.Sprintf("(%s = ?)", searchattribute.GetSqlDbColName(searchattribute.NamespaceID)),
+	)
+	queryArgs = append(queryArgs, namespaceID.String())
+
+	if len(queryString) > 0 {
+		whereClauses = append(whereClauses, fmt.Sprintf("(%s)", queryString))
```

Comment:
Do we have sql injection risk here?

Reply:
It shouldn't. The query converter escape the user's input.